Tasks/Anastasia Konuyk/Task2/Task2.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from num2words import num2words as n2w
+
+current_time = datetime.datetime.now().strftime('%H:%M').split(':')
+
+def result(hours, minutes):
+    if minutes == 0:
+        return(f'{n2w(hours-12)} o\'clock') if hours > 12 else (f'{n2w(hours)} o\'clock')
+    elif minutes < 30 or 30 < minutes < 45:
+        return(f'{n2w(minutes)} minutes past {n2w(hours-12)}') if hours > 12\
+            else(f'{n2w(minutes)} minutes past {n2w(hours)}')
+    elif minutes == 30:
+        return (f'half past {n2w(hours - 12)}') if hours > 12 else (f'half past {n2w(hours)}')
+    else:
+        return (f'{n2w(60-minutes)} minutes to {n2w(hours-11)}') if hours > 12\
+            else (f'{n2w(60-minutes)} minutes to {n2w(hours+1)}')
```

Comment:
that's good and all, but it goes straight against the task description, it should be russion time definitions, -20 points for the missing part of the task